test/common/vmmanage.py

```diff
@@ -66,7 +67,7 @@ def build_and_maybe_install(image, do_install=False, skip=None, args=None):
         finally:
             machine.stop()
 
-def only_install(image, skip=None, args=None, address=None):
+def only_install(image, skips=[], args=None, address=None):
```

Comment:
Mutable types as a default argument for a kwarg are usually a bad idea. Better to use None and instantiate to a blank value in the code.